js/src/forum/components/Page.js

```diff
@@ -1,33 +1,14 @@
-import Component from '../../common/Component';
+import Page from '../../common/components/Page';
 
 /**
- * The `Page` component
+ * The `ForumPage` component
  *
  * @abstract
  */
-export default class Page extends Component {
+export default class ForumPage extends Page {
   init() {
-    app.previous = app.current;
-    app.current = this;
+    super.init();
```

Comment:
Why are we still overriding this method (or this class, for that matter)? The `Page` class in `common` has the `app.modal.close()` line as well.

Reply:
The two classes were equal before (except for the `app.drawer.hide()`, which may have been an oversight), so combining them into one may make sense. On the other hand, it seems to be a BC break for no particularly important reason, and it seems at least possible that we'd want to have separate base classes for admin and forum pages at some point in the future...

Reply:
I meant to put app.drawer.hide, not app.modal.close. That's my bad. As to pages, unless we're planning on adoping a radically different structure, (no modals and no PageState on admin), I'd say having one central class would be good. To keep BC, we could add an AdminPage class that extends off of Page and doesn't do anything